app/models/event.rb

```diff
@@ -110,7 +110,6 @@ def standing_of(team)
   end
 
   # this is a method that simplifies manual testing, not intended for production use
-  # method not used at the moment since it is now testet with joined users
```

Comment:
I don't know why you do that here

Reply:
Somehow the merge went terribly wrong :)